pegasus/sites.v3/code.org/views/breadcrumbs.haml

```diff
@@ -0,0 +1,13 @@
+%div.breadcrumbs
+  - trail.each_with_index do |crumb, index|
+    - if !crumb[:url] || index == trail.size - 1
+      %strong.crumb.active
+        #{crumb[:text]}
+    - else
+      %a.crumb.inactive{href:"#{crumb[:url]}"}
+        #{crumb[:text]}
+      >
+:css
+  .active {
```

Comment:
That's an even better idea (that way something else called `.active` wouldn't affect this in the future)